src/server/handle-redirects.js

```diff
@@ -4,34 +4,35 @@ import fetch from 'isomorphic-fetch'
 import { log } from '../utilities/logger'
 
 const setRedirects = (server, redirects) => {
-  Object
-    .keys(redirects)
-    .forEach(fromPath => {
-      server.route({
-        method: 'GET',
-        path: `${fromPath}`,
-        handler: (request, reply) => {
-          reply
-            .redirect(`${redirects[fromPath]}${request.url.search}`)
-            .permanent()
-            .rewritable(true)
-        }
-      })
-    })
+  server.ext('onPostHandler', (request, reply) => {
+    const status = reply.request.response.statusCode
+    if (
+      (status == 404)
+      && redirects
+      && (request.url.pathname in redirects)
```

Comment:
How performant is this? Are there faster methods?

Reply:
What's the `in` syntax? Never seen it before – is it a new JS feature?

Reply:
It's certainly ES5, maybe even ES3!

Reply:
`key in object` basically

Reply:
wow – learned something new - before my time!

Reply:
It's probably negligible, but `typeof redirects[request.url.pathname] !== undefined` seems quicker in this benchmark https://jsperf.com/array-indexof-vs-object-s-in-operator/4